vms/platformvm/txs/executor/standard_tx_executor.go

```diff
@@ -54,13 +54,20 @@ func (e *StandardTxExecutor) CreateChainTx(tx *txs.CreateChainTx) error {
 		return err
 	}
 
+	var (
+		timestamp       = e.State.GetTimestamp()
```

Comment:
nit: `timestamp` -> `currentTimestamp` to match the usage in other funcs

Reply:
Done